### DISCUSSION TITLE
BALANCEO (ROLL). NO FUNCIONA EN ETS2

### DISCUSSION THREAD

**Discussion by anjelooo** [opentrack/opentrack#1810] [General]
saludos a todos. Estoy usando opentrack y aitrack. en ets2 v.1.48 y en balnceo (roll) no me funciona. como en el vídeo. https://www.youtube.com/watch?v=9JTjaTWX5j8si me porian dar una mano.

**Comment by Pieloth** [comment]
Roll is not used by ETS2 and ATS. Only Yaw and Pitch are used
Not an issue IMHO anyway

**Reply by anjelooo** [reply]
hola gracias por responder. lo que quiero decir. es que lo quiero usar en el juego ets2. no digo que no me guste. en el video de prueba de youtube. lo hace fantastico. pero en el mio no. no logro que me funcione. ahora le estoy cambiano los controladores a la camara ps3. noce si ese sea el problema.

**Comment by Pieloth** [comment]
Yes it works fine on video, but this is how it should actually work on any setup with AItrack, or simply Neuralnet tracker
Did you try with a basic webcam first, before going to PS3 ?

**Reply by anjelooo** [reply]
con una camara web basica no. lo intentare. puede que sea ese el problema?. conseguire una e intentare. si persiste el problema te comunicare gracias por tu tiempo.

**Comment by Pieloth** [comment]
PS3 uses a specific driver. A basic Webcam will use Windows native driver, it will be easier to check if it's working.
If it's OK then you can focus on PS3

**Reply by anjelooo** [reply]
saludos pieloth. he probado con una camara web basica, pero no logro hacer que funcione el balanceo (roll).
esta es la configuracion que usan en la prueba del video en youtube.
 
Head Tracking:
AITrack: [https://github.com/AIRLegend/aitrack#...](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbnlrU1BXLVRqRk1WSGZtYVRxUENVaTIzeExLUXxBQ3Jtc0ttamRMOXF2dmZoR3Q4clFqQTM4d1hObzF6QUlrNXRFUllFQU9ibWpmTTg2NUV2X1lyQ2hjVEtBSUxqaTd1UXJ3Y3hnNjZhS2VpM1pVU0U2clJ3all5LWVGNktKMHVjSEhCTU5oelljMG1BZzZiRDBNTQ&q=https%3A%2F%2Fgithub.com%2FAIRLegend%2Faitrack%23readme&v=9JTjaTWX5j8)
OpenTrack: [https://github.com/opentrack/opentrac...](https://www.youtube.com/redirect?event=video_description&redir_token=QUFFLUhqbklEMWJPTFZIYkRHMTBfS2dyZzJTbnhXNjBtZ3xBQ3Jtc0tuMFhReUEyMlp0c2l2dFc0MzE2SFdFcFpiVF9ieG9XOHExaGJDenY4d2JWb2tpY2E2VXBoRUN1UXh5b3RaRVdMdzJNbk40TXN3TS1FS1pqYW44NlRYZmxhZnZLMW9SejEwbzFaRlpBSkdvSEZEdVVnYw&q=https%3A%2F%2Fgithub.com%2Fopentrack%2Fopentrack%23readme&v=9JTjaTWX5j8)
PS3 Eye

Gamepad: Logitech Cordless Rumblepad 2

PC Spec:
AMD Ryzen 5 2600 4.0GHz (OC)
AMD RX570 4GB [MSI Armor] 1320/2100 OC
2x4 GB RAM 3200MHz CL 15
SSD PNY XLR8 CS3030 250GB + Crucial BX500 1TB + HDD Seagate 1TB

All setting to ultra (r_ssao "3")

si me podrias ayudar te lo agradeceria mucho. al no saber mucho de programacion, se me complica un poco.
en el video usan aitrack, opentrack y ps3. ya hice de todo instale y desintale el juego ets2. miles de veces. nose dande estara el problema. usaran otro codigo mas. desde ya muchas gracias.

**Comment by Pieloth** [comment]
I don't have roll either in ETS2 or ATS, using OpenTrack with Neuralnet or AItrack. 
So I don't know which device they use in the video to get roll in ETS2
However, roll works fine in MSFS2020 for instance. 
So I'm pretty sure ETS2 disables roll by default, maybe it's possible to activate it, but I don't know how (and I would not use it anyway)

**Reply by anjelooo** [reply]
gracias por tu respuesta. estuve viendo que si funciona el roll en MSFS2020. lastima me hubiese gustado mucho usar el roll en ETS2. le da un poco mas de realismo al juego. gracias por responder, te mando un abrazo desde argentina.

**Comment by Pieloth** [comment]
A quick google search for "ets2 tracker roll" shows immediate answer:
[https://steamcommunity.com/app/227300/discussions/0/1742229167205219440/](https://steamcommunity.com/app/227300/discussions/0/1742229167205219440/)
Change a setting in your profile controls.sii file

**Reply by anjelooo** [reply]
esto vendria a hacer en el juego verdad?

**Reply by anjelooo** [reply]
gracias pieloth ese era el verdadero poblema. eres un genio. la verdad que te lo re agradesco. muchas pero muchas gracias.

**Comment by Pieloth** [comment]
No No
This is a file to edit manually, which is in your profile files for this game
Probably located in your folder:
\<my documents>\Euro Truck Simulator 2\steam_profiles\\\<Profile number>

**Reply by anjelooo** [reply]
si, si ya lo encontre. nuevamente gracias. que tengas un lindo dia. abrazos desde argentina.

**Comment by Pieloth** [comment]
👍
